Documentation/preparing-devel-environment.md

```diff
@@ -0,0 +1,90 @@
+# Preparing development environment and first rkt build
+This is example configuration for fresh instalation of Ubuntu 16.04 GNOME.
+
+### Required software
+You have to install few software packages to correctly build:
+```sh
+$ sudo apt-get install libssl-dev dh-autoreconf libacl1-dev libtspi-dev build-essential
```

Comment:
I think this is another build-deps list to be maintained and can quickly get out-of-date. We have a similar thing for CI machines under `scripts/install-deps-debian-sid.sh`. It is tailored at Debian unstable, but I think it will keep working also for recent Ubuntu.
Can you please check if it contains all the build-deps you need, and then change here to just call that script?